src/Emulator/Main/Peripherals/CPU/ICpuSupportingGdb.cs

```diff
@@ -5,9 +5,39 @@
 // Full license text is available in 'licenses/MIT.txt'.
 //
 using System;
+using System.Collections.Generic;
 
 namespace Antmicro.Renode.Peripherals.CPU
 {
+    public struct IGBDRegister
```

Comment:
This is a struct so its name should not start with `I` (as this is, by convention, reserved for interfaces).

```suggestion
    public struct GBDRegisterDescriptor
```